aria/CHANGELOG.md

```diff
@@ -5,6 +5,15 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 0.2.0 (UNRELEASED)
+### Changed
+- Bump `cipher` dependency to v0.5
```

Comment:
I am not sure what PR we should link here. We either have to list all pre-release updates, or only the final update to the full release of cipher v0.5.